### ISSUE TITLE
A11y:Interactive component must have a programmatically associated name on slider's thumb

### ISSUE THREAD

**Issue by maohuang81** [carbon-design-system/carbon#7059]
### Environment

macOS Mojave version 10.15.4
Chrome Version Version 86.0.4240.75
Carbon v10 - React
IBM Equal Access Accessibility Checker 3.1.0

### Detailed Description
![image](https://user-images.githubusercontent.com/63094956/96248817-be03ab80-0fde-11eb-81aa-68c34a96ac9c.png)

### Expected Result
There is an aria-label for slider's thumb, as same as slider's input.

**Comment by maohuang81** [comment]
Hi @joshblack Do we have any update? Thanks in advance.
